types/withdrawal.yaml

```diff
@@ -0,0 +1,12 @@
+Withdrawal:
+  type: object
+  properties:
+    index:
```

Comment:
it looks like the current spec also has `validator_index` here 
https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/beacon-chain.md#withdrawal

Reply:
Added; thanks.